src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
```

Comment:
These two variables `logo` and `body` are declared at the top of the file and are not going to be changed. It would be better to use `const` for these variables to avoid accidental reassignment.